content/case-studies/mercedes-benz.md

```diff
@@ -29,11 +29,13 @@ sections:
       anchor: developers-in-the-driver-s-seat-programming-the-cloud-with-pulumi
     - label: Conclusion
       anchor: the-road-ahead
-      
+
 aliases:
     - /case-studies/merceded-benz
 ---
 
+{{< youtube "MGA_Bba2GqQ" >}}
```

Comment:
Should this have `?rel=0` per https://github.com/pulumi/docs/blob/master/BLOGGING.md#video ?

```suggestion
{{< youtube "MGA_Bba2GqQ?rel=0" >}}
```